content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -0,0 +1,216 @@
+* External dependencies have been minimized (just a modern kernel and cgroup mounts needed). K3s packages required dependencies, including:
+  * containerd
+  * Flannel
+  * CoreDNS
+  * CNI
+  * Host utilities (iptables, socat, etc)
+  * Ingress controller (traefik)
+  * Embedded service loadbalancer
+  * Embedded network policy controller
+
+The Windows Subsystem for Linux lets developers run a GNU/Linux environment -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a traditional virtual machine or dualboot setup.
+
+1. Install WSL2 as per official Microsoft Documentation [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
+2. Update to WSL2 and download your favorite distibution:
+     * [SUSE Linux Enterprise Server 15 SP1](https://www.microsoft.com/store/apps/9PN498VPMF3Z)
+     * [openSUSE Leap 15.2](https://www.microsoft.com/store/apps/9MZD0N9Z4M4H)
+     * [Ubuntu 18.04 LTS](https://www.microsoft.com/store/apps/9N9TNGVNDL3Q)
+     * [Debian GNU/Linux](https://www.microsoft.com/store/apps/9MSVKQC78PK6)
+
+**References**:
+
+* [K3S on WSL2 - Quick Start Guide](https://gist.github.com/ibuildthecloud/1b7d6940552ada6d37f54c71a89f7d00)
+
+### Create your cluster on K3S on WSL2
+
+Creating a Kubernetes cluster is as simple as `sudo ./k3s server`.
+
+This will bootstrap a Kubernetes cluster but you will cannot yet access from your windows 10 machine to the cluster itself.
+
+**Note:** You can't install k3s using the curl script because there is no supervisor (systemd or openrc) in WSL2.
+
+* Download k3s binary from https://github.com/rancher/k3s/releases/latest
+
+`chmod +x k3s`
+
+* Run k3s
+
+`sudo ./k3s server`
+
+### Setup access to your WSL2 instance
+
+* Copy `/etc/rancher/k3s/k3s.yaml` from WSL to your home in Windows to `%HOME%.kube\config`. Edit the copied file and change the server URL from `https://localhost:6443` to the IP of the your WSL2 instance (`ip addr show dev eth0`). So something like `https://192.168.170.170:6443`.
+* Run kubectl from windows, if you don't have it yet you may obtain it from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows)
+
+## Deploying Kubeflow Pipelines
+
+The installation process for Kubeflow pipelines is the same for all the environments: kind, k3s, k3s on WSL2.
+
+1. Deploy the Kubeflow Pipelines:
+
+```
+export PIPELINE_VERSION={{% pipelines/latest-version %}}
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
+kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/platform-agnostic-pns-executor?ref=$PIPELINE_VERSION"
```

Comment:
note the PR actually added it as `platform-agnostic-pns`, without executor

Reply:
fixed